.github/workflows/build.yml

```diff
@@ -26,8 +26,12 @@ jobs:
     - name: Install Python dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install coverage
-        pip install -r requirements.txt
+        pip install .[test]
+        pip install .[mysql,knx,dmx,midi,mqtt,pulse,telegram,file_persistence]
+        pip uninstall python-rtmidi -y
```

Comment:
This is a workaround so some midi tests are skipped as in the main branch of the core reps.  See PR description for details.